test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersionsComprehensive.g.cs

```diff
@@ -26,6 +26,7 @@ public class PackageVersionsComprehensive
 #if DEFAULT_SAMPLES
                 new object[] { string.Empty },
 #else
+#if NETFRAMEWORK
```

Comment:
The changes to `MinVersionNetCore` take effect here, where we do not run the XUnit test cases for package versions that are lower than the specified `MinVersionNetCore`